FluentAssertions.sln

```diff
@@ -145,8 +145,8 @@ Global
 		{FCAFB0F1-79EA-4D49-813B-188D4BC4BE71}.Release|Any CPU.Build.0 = Release|Any CPU
 		{05727385-DA56-4FC1-B44A-79DFECC3198A}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{05727385-DA56-4FC1-B44A-79DFECC3198A}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{05727385-DA56-4FC1-B44A-79DFECC3198A}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{05727385-DA56-4FC1-B44A-79DFECC3198A}.Release|Any CPU.Build.0 = Release|Any CPU
+		{05727385-DA56-4FC1-B44A-79DFECC3198A}.Release|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
Why do you build "Debug" with Release configuration?

Reply:
Otherwise [Subject Identification](https://fluentassertions.com/documentation/#subject-identification) won't work.

Reply:
I think, instead the project configuration should be changed to not optimize.
Otherwise it is confusing that there is a Release configuration which is never used.

Reply:
We could do that, but now with this PR.